gomp-git-PKGBUILD/PKGBUILD

```diff
@@ -0,0 +1,64 @@
+#
+# Maintainer: Mahmoud Mohamed (00xWolf) <mmsaeed509@gmail.com> , <https://github.com/mmsaeed509>
+#
+
+pkgname=gomp-git
+pkgver=r329.0c0cbbf
+pkgrel=3
+pkgdesc='MPD client inspired by ncmpcpp written in GO.'
+arch=('any')
+url="https://github.com/aditya-K2/gomp"
+license=('GPL3')
+
+makedepends=(
+  'git'
+   'go'
+   )
+
+source=(
+  "${pkgname}::git+${url}"
+  "config.yml"
+  "cover.jpg"
+  )
+
+sha256sums=(
+  'SKIP'
+  'SKIP'
+  'SKIP'
+  )
+
+groups=('Exodia-TUI-Apps')
+install=${pkgname}.install
+
+pkgver() {
+
+  cd ${pkgname}
+  printf "r%s.%s" "$(git rev-list --count HEAD)" "$(git rev-parse --short HEAD)"
+
+}
+
+prepare() {
+
+   cp config.yml ${pkgname}
+   cp cover.jpg ${pkgname}
+   cd ${pkgname}
+   mkdir -p build
+
+}
+
+build() {
+
+    export GOFLAGS="-buildmode=pie -trimpath -mod=readonly -modcacherw"
+    cd ${pkgname}
+    go build -o build
+
+}
+
+package() {
+
+    cd ${pkgname}
+    install -D build/gomp -t "${pkgdir}/usr/bin/"
+    install -D config.yml -t "${pkgdir}/etc/skel/.config/gomp"
```

Comment:
Where are the config.yml and cover.jpg files? 
They are @ [Exodia-TUI-Apps](https://github.com/Exodia-OS/Exodia-TUI-Apps/tree/master/gomp-git) but not in the `gomp` repo. So can you change it to copy the `sample_config.yml`. And about `cover.jpg` maybe I will add a default image to the extras folder Can you copy that instead?

Reply:
I'm using `sample_config.yml` I renamed it to `config.yml` and changed the values of:
```yml
ADDITIONAL_PADDING_X : 20
ADDITIONAL_PADDING_Y : 23

IMAGE_WIDTH_EXTRA_X  : -1.2
IMAGE_WIDTH_EXTRA_Y  : -3
```
to render cover.jpg

I'll update the PR